socs/common/moxa_serial.py

```diff
@@ -1,3 +1,5 @@
+# 9/2025 Shun
+# Change all BaseExeption --> TimeoutError
```

Comment:
Unless you feel strongly about including this changelog in the file, I'd suggest we just rely on `git blame` for this sort of detail.

This code was imported from an older project "as is", which is the only reason this old comment was left here.